src/app/classes/Status.ts

```diff
@@ -9,6 +11,10 @@ export function getStatusName(status: Status): string {
         return 'Fire';
     case Status.Advantage:
         return 'Advantage';
+    case Status.MindControl:
+        return 'Hypnotized';
+    case Status.Armored:
+        return 'Armored';
     }
 }
```

Comment:
i would probably use a multiton here so that the name and emoji can be members of the statuses rather than needing a separate function to look them up

Reply:
This has been done; you might want to check and see if I've done it right.